### PR TITLE
Add fishbrain_id strategy

### PR DESCRIPTION
**Pull request by alexceder**

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# OmniAuth Fishbrain
+
+OmniAuth strategy for Fishbrain
+
+## Installation
+
+```ruby
+gem 'omniauth-google-oauth2'
```
Comment (dalen) [inline README.markdown]: This must be a copy+paste error

File: lib/omniauth/strategies/fishbrain.rb [hunk 2]
```diff
@@ -1,84 +1,58 @@
...
-             token_url: '/oauth2/token',
-             auth_scheme: :basic_auth
+      option :client_options, site: 'https://accounts.fishbrain.com',
+                              authorize_url: '/oauth2/authorize',
+                              token_url: '/oauth2/token',
+                              auth_scheme: :basic_auth
       option :scope, 'email openid profile'
-      option :jwt_leeway, 60
-      option :user_pool_id, 'eu-west-1_5r0WbR8OH'
+      option :user_pool_id, 'eu-west-1_K2uP41DlP'
```
Comment (dalen) [inline lib/omniauth/strategies/fishbrain.rb]: This is the staging pool id. The default value should be the production one.

File: lib/omniauth/strategies/fishbrain_id.rb [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+module OmniAuth
+  module Strategies
+    class FishbrainId
+      include OmniAuth::Strategy
+      include OmniAuth::Fishbrain::VerifiesIdToken
+
+      args %i[user_pool_id client_ids aws_region]
+
+      option :name, 'fishbrain_id'
+      option :user_pool_id, nil
```
Comment (dalen) [inline lib/omniauth/strategies/fishbrain_id.rb]: This could also have the prod ID as default

File: README.markdown (hunk 1 above)
Comment (alexceder) [reply README.markdown]: lol, that was the initial reason I changed the readme, then forgot about the main point. 🐰

**Review by dalen — APPROVED** [review]
Other than the things I commented about it looks good to me

**Pull request merged**